db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,12 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
+      t.string :labeled_resource_type
+      t.string :label_name
+      t.text :label_value
+      t.belongs_to :tag, :type => :bigint
```

Comment:
Too late as you already merged?  Let me know if you want a followup PR for any of your comments.